webpack/JobWizard/__tests__/integration.test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import { mount } from '@theforeman/test';
+import * as api from 'foremanReact/redux/API';
+import { JobWizard } from '../JobWizard';
+import * as selectors from '../JobWizardSelectors';
+
+jest.spyOn(api, 'get');
+jest.spyOn(selectors, 'selectJobTemplate');
+jest.spyOn(selectors, 'selectJobTemplates');
+jest.spyOn(selectors, 'selectJobCategories');
+jest.spyOn(selectors, 'selectJobCategoriesStatus');
```

Comment:
just wondering if there is a way to pass a mocked store with this values so the selectors would work the same and grab what they need from the predefined store.

Reply:
I mocked the selectors twice in the test, once as their initial state, then click something, check that the correct action was called, and mock the selectors a second time with the response